XamlControlsGallery/Common/VariedImageSizeLayout.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+using VirtualizingLayout = Microsoft.UI.Xaml.Controls.VirtualizingLayout;
+using VirtualizingLayoutContext = Microsoft.UI.Xaml.Controls.VirtualizingLayoutContext;
+
```

Comment:
I think you can safely remove these.

Reply:
It throws a bunch of errors when I remove these definitions (because I'm not using the Microsoft.Ui.Xaml in this file)?